model/recipesModel.js

```diff
@@ -1,14 +1,16 @@
-const db = require('../data/db-config');
+const DB = require("../data/db-config");
 
 module.exports = {
-  find,
+  getChefRecipes,
   findById,
   add,
   update,
   remove
 };
 
-function find() {}
+function getChefRecipes(id) {
+  return DB("recipes").where("chef_id", "=", `${id}`);
```

Comment:
There is no need for string interpolation on the id. You can simply pass the argument into .where.
`.where("chef_id", "=", id)`
Also, `.where("chef_id", id)` might work.